CHANGELOG.rst

```diff
@@ -1,6 +1,60 @@
 Changelog
 =========
 
+1.6.0 (2022-06-28)
+------------------
+
+New Operators
+"""""""""""""
+
+This release adds the following 4 new async sensors/operators:
+
+.. list-table::
+   :header-rows: 1
+
+   * - Operator/Sensor Class
+     - Import Path
+     - Example DAG
+
+   * - ``DataprocCreateClusterOperatorAsync``
+     - .. code-block:: python
+
+        from astronomer.providers.google.cloud.operators.dataproc import DataprocCreateClusterOperatorAsync
+     - `Example DAG <https://github.com/astronomer/astronomer-providers/blob/main/astronomer/providers/google/cloud/example_dags/example_dataproc.py>`__
+
+   * - ``DataprocDeleteClusterOperatorAsync``
+     - .. code-block:: python
+
+        from astronomer.providers.google.cloud.operators.dataproc import DataprocDeleteClusterOperatorAsync
+     - `Example DAG <https://github.com/astronomer/astronomer-providers/blob/main/astronomer/providers/google/cloud/example_dags/example_dataproc.py>`__
+
+   * - ``DataprocUpdateClusterOperatorAsync``
+     - .. code-block:: python
+
+        from astronomer.providers.google.cloud.operators.dataproc import DataprocUpdateClusterOperatorAsync
+     - `Example DAG <https://github.com/astronomer/astronomer-providers/blob/main/astronomer/providers/google/cloud/example_dags/example_dataproc.py>`__
+
+   * - ``RedshiftDeleteClusterOperatorAsync``
+     - .. code-block:: python
+
+        from astronomer.providers.amazon.aws.operators.redshift_cluster import RedshiftDeleteClusterOperatorAsync
+     - `Example DAG <https://github.com/astronomer/astronomer-providers/blob/main/astronomer/providers/amazon/aws/example_dags/example_redshift_cluster_management.py>`__
+
+
+Enhancements
+""""""""""""
+
+- Implement OpenLineage custom extractor for BigQuery Async Operators
+  (`#429 <https://github.com/astronomer/astronomer-providers/pull/429>`_)
+- Add session specific query tag and OpenLineage Extractor for Snowflake Async operator
+  (`#437 <https://github.com/astronomer/astronomer-providers/pull/437>`_)
+
```

Comment:
This was also an improvement in dataproc hook. though, I'm not sure if we need to put it here or not https://github.com/astronomer/astronomer-providers/pull/453

Reply:
That doesnt change the way users use the dataproc submit operator, its more of a technical change. Hence in my opinion, it doesnt need to be included

Reply:
We can include https://github.com/astronomer/astronomer-providers/pull/448 as an improvement

Reply:
included in [db210fb](https://github.com/astronomer/astronomer-providers/pull/480/commits/db210fb370f0f088e01ad7fd9c785196eed831fd)